docs/concepts/workers.md

```diff
@@ -29,6 +29,8 @@ See example Worker code for:
 - [PHP SDK](/docs/php/workers)
 - [Node SDK](/docs/node/hello-world/#worker)
 
+Note that [Temporal also supports a polyglot programming model](https://github.com/tsurdilo/temporal-polyglot) because you can signal external workflows. This means that workflows can be run independently of teams maintaining microservices in different languages.
```

Comment:
```suggestion
Note that [Temporal also supports a polyglot programming model](https://github.com/tsurdilo/temporal-polyglot) because you can start workflows by string name and send signals between workflows.
This means that workflows can be run independently of teams maintaining microservices in different languages.
```